$MirOS: ports/lang/pcc/patches/patch-cc_cc_cc_1,v 1.1 2008/11/10 01:53:13 tg Exp $
--- cc/cc/cc.1.orig	Mon Apr  5 14:23:42 2010
+++ cc/cc/cc.1	Sat Apr  2 13:01:05 2011
@@ -157,6 +157,8 @@ to create debug output.
 This unsets the
 .Fl O
 option.
+Debug information output can be disabled with
+.Fl g0 .
 .It Fl I Ar path
 Passed to the
 .Xr cpp 1
@@ -219,6 +221,8 @@ to
 Note: this is unset if the
 .Fl g
 option is used.
+Optimisations can be disabled using
+.Fl O0 .
 .It Fl o Ar outfile
 Save result to
 .Ar outfile .
